src/main/java/core/basesyntax/util/filereader/FileReaderImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.util.filereader;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String fromFileName) {
```

Comment:
It's recommended to use try-with-resources to ensure that all resources are closed properly. In this case, the Files.readAllLines method does not require it since it closes resources internally, but it's a good practice to be aware of for other I/O operations.